### ISSUE TITLE
Collector Limpet Controller - Failure to Identify 

### ISSUE THREAD

**Issue by awp3rator** [EDCD/EDDI#1551]
## EDDI version in which issue is found
[Please give the full version number here, not just "latest" or "current"]
3.4.3

## VoiceAttack version in which issue is found (as applicable)
[Skip this section if VoiceAttack was not running when the issue occurred. Please give the full version number here, not just "latest" or "current"]
N/A - VA not running

## Steps to reproduce
1. Federal Corvette with class 5 collector limpet controller installed, docked at a station
2. Have 1 or more limpets on board (testing with 72)
3. Launch from station
4. Listen for EDDI limpet check message

## Expected
Nothing, the ship has both a collector limpet controller and adequate limpets to pass the low stock check.

## Observed
EDDI announces that the ship has limpets but no limpet controller 

## Investigation
Generated EDDI's log file and inspected the ShipMonitor:HandleProfile.  Ship has "int_dronecontrol_collection_size5_class5" installed, but no "controller".  eddi.json appears to be looking for "controller".

EDDI logs attached
[eddi_issue.zip](https://github.com/EDCD/EDDI/files/3693453/eddi_issue.zip)

**Comment by awp3rator** [comment]
For whatever reason, removing the module and reinstalling it in the ship seemed to make it as intended.  I no longer the warning when the above conditions are met, and I correctly get warnings when:

1. Less than 10% of cargo hold is limpets - warning about restocking 
2. No limpets - warning I have a controller but no limpets (when carrying no limpets)
3. No controller - warning I have limpets but no controller (when controller not installed)

**Comment by Tkael** [comment]
There's a null value exception we've seen a couple of times with the latest released version that can cause EDDI to drop / not update correctly from the `Loadout` event. 
It looks like you hit it. A fix is already in place for the next release. o7

**Comment by awp3rator** [comment]
Full disclosure, I had changed ships a few times and module load outs before this happened.  This all happened after not playing for a bit, updating the game to 3.0+, EDDI from 2.4.something 3.4.3 and playing for a bit.  I have also noticed EDDI commenting that I have bounties when I don't...  I had valid bounties, but then cleared them.  I will open a separate ticket if I can pinpoint when/how that issue occurs.
